app/src/utils/LoadingPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {useEffect} from 'react';
+import {useParams, useNavigate} from 'react-router-dom';
+
+const LoadingPage: React.FC = () => {
```

Comment:
Could you do export const instead of having the export default at the bottom?

Reply:
Fixed